src/index.ts

```diff
@@ -1,36 +1,25 @@
 // tslint:disable-next-line: no-reference
 /// <reference path="../types/dom-monkeypatch.d.ts" />
 
-/* tslint:disable:no-var-requires */
-const version = require("../package.json").version;
-
 // Expose all types
-export * from "./horizon_api";
-export * from "./server_api";
+export * from './errors';
+export { Config } from './config';
+export { Utils } from './utils';
 
-// stellar-sdk classes to expose
-export * from "./account_response";
-export * from "./errors";
-export { Config } from "./config";
-export { Server } from "./server";
-export {
-  FederationServer,
-  FEDERATION_RESPONSE_MAX_SIZE
-} from "./federation_server";
-export {
-  StellarTomlResolver,
-  STELLAR_TOML_MAX_SIZE
-} from "./stellar_toml_resolver";
-export {
-  default as HorizonAxiosClient,
-  SERVER_TIME_MAP,
-  getCurrentServerTime
-} from "./horizon_axios_client";
-export * from "./utils";
+// TOML and federation resolvers to expose
+export * as Federation from './federation';
 
-// expose classes and functions from stellar-base
-export * from "stellar-base";
+// Horizon-related classes to expose
+export * as Horizon from './horizon';
+
+// Soroban RPC-related classes to expose
+export * as SorobanRpc from './soroban';
+export { ContractSpec } from './soroban'; // not RPC related, so at top-level
```

Comment:
does this mean there will be `SorobanRpc.ContractSpec` and `ContractSpec` ? if so, for consider, remove it from export in ./soroban/index.ts and just export it here instead:

```
export { ContractSpec as SorobanContract} from './soroban/contract_spec';
```

Reply:
good point, I just moved the file to the top-level path since it doesn't need to be in `src/soroban`